examples/00-EDB/02_edb_to_ipc2581.py

```diff
@@ -1,7 +1,7 @@
 """
-EDB: IPC2581 export
+EDB: ipc2581 export
```

Comment:
Ansys and Google style guidelines both say to use all capital letters for file extensions when they are not part of a reference to a full file name, such as "test.ipc2581" Consequently, many of the edits made changing the extensions to lower case letters should be rejected.